fabtests/configure.ac

```diff
@@ -185,13 +188,7 @@ AS_IF([test x"$with_ze" != x"no" && test -n "$with_ze" && test "$have_ze" = "0"
 AC_DEFINE_UNQUOTED([HAVE_LIBZE], [$have_ze], [ZE support])
```

Comment:
Nitpicky but can we rename HAVE_LIBZE to HAVE_ZE? We are no longer checking for the whole library.